.travis.yml

```diff
@@ -18,6 +18,7 @@ install:
 before_script:
   - psql -c 'create database travis_ci_test;' -U postgres
   - cd oshc/
+  - python manage.py makemigrations --noinput
```

Comment:
@vaibhavsingh97 Let's fix this how Jarif has suggested it in #84. Plus this has two commits which fixes two different issues. Cleanup?